### DISCUSSION TITLE
Laravel LogSnag

### DISCUSSION THREAD

**Discussion by hosmelq** [HassanZahirnia/laravel-package-ocean#123] [Package Suggestions]
Easily integrate [LogSnag](https://logsnag.com/)'s event tracking into your Laravel app. Monitor user activity, get realtime analytics, and receive instant insights.

https://github.com/hosmelq/laravel-logsnag

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
